GliaWidgets/View/Chat/ChatView.swift

```diff
@@ -92,8 +98,6 @@ class ChatView: EngagementView {
         tableView.backgroundColor = .clear
         tableView.delegate = self
         tableView.dataSource = self
-        tableView.rowHeight = UITableView.automaticDimension
-        tableView.estimatedRowHeight = UITableView.automaticDimension
```

Comment:
In case when `content(for item:)` returns `.none` case and `ChatItemCell` removes all subviews, so there is no content, tableView applies for this cell 44point height. So instead of to hide cell, there is blank space. To avoid this, I had to realise `estimatedHeightForRow` delegate method and return zero height for `.none` ChatItem.